articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to create a custom image from a VM on Azure Stack - Portal | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using the portal
+services: 
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM - Portal
+toc_fullpath: Users/How To/azs-how-create-vm-image-portal.md
+toc_mdlink: azs-how-create-vm-image-portal.md
+---
+
+# How to create a custom image from a VM on Azure Stack - Portal
+
+## Overview
+
+An image resource can be created from a generalised virtual machine (VM) that is stored as either a managed disk or an unmanaged disk in a storage account. The image can then be used to create multiple VMs.
+
+This article will explain how to create a custom image from a VM on Azure Stack, which you can then use to deploy other VMs.
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the **Azure Stack** portal.
+
+## [1. Generalise your VM](#tab/tabid-1)
+
+> [!WARNING]
+> Once a VM has been generalised, you **can not** log back into it.
+
+## [2. Creating the image - Portal](#tab/tabid-2)
+
+> [!WARNING]
+> Capturing a VM image will make it unusable and cannot be undone.
+
+1. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com/).
+
+2. Click **Virtual Machines** on the left menu.
+
+3. Click the VM that you want to capture.
+
+4. At the top, click **Capture**.
+
+    ![Capture VM image](images/azs-browser-capture-vm.png)
+
+5. In the *new* blade, provide a **Name** for the new image.
+
+6. Select a **Resource Group** from the dropdown menu or create a new one.
```

Comment:
**Resource group**

(it's a minor thing, but try to match the labels exactly as they appear in the UI, including capitalisation)